DataFormats/Luminosity/interface/PixelClusterCounts.h

```diff
@@ -0,0 +1,56 @@
+#ifndef DataFormats_Luminosity_PixelClusterCounts_h
+#define DataFormats_Luminosity_PixelClusterCounts_h
+/** \class reco::PixelClusterCounts
+ *  
+ * Reconstructed PixelClusterCounts object that will contain the moduleID, BX, and counts.
+ *
+ * \authors: Sam Higginbotham shiggib@cern.ch and Chris Palmer capalmer@cern.ch
+ * 
+ *
+ */
+#include <algorithm>
+#include <string>
+#include <sstream>
+#include <iostream>
+#include <vector>
+
+namespace reco {
+class PixelClusterCounts {
+static constexpr unsigned int nEmBX=3564;//Empty BX to fill with counts
+    public:
+        PixelClusterCounts() : m_events(nEmBX){}
+        ////////////////////////////////////////////
+        void increment(int mD,int BXid,int count){
+            std::vector<int>::iterator it;
+            it = std::find(m_ModID.begin(), m_ModID.end(), mD);
+            size_t modIndex = it - m_ModID.begin();
+
+            if(it == m_ModID.end()){
+                std::vector<int> m_empBX(nEmBX, 0);
+                m_ModID.push_back(mD);
+                m_counts.insert(m_counts.begin(),m_empBX.begin(),m_empBX.end()); 
```

Comment:
the count increment is done below for the module at index corresponding to the end of the vector module, why is this insert made in front?
It looks incorrect.
My suggestion above should probably replace this